docs/curl/provenance.md

```diff
@@ -0,0 +1,829 @@
+    "header": {
+      "node_id": "bdb-node-1"
+    },
+    "values": [
+      {
+        "value": "eXl5",
+        "metadata": {
+          "version": {
+            "block_num": 4
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true,
+              "bob": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "value": "dGhpcyBpcyBhIGZpcnN0IHZhbHVl",
+        "metadata": {
+          "version": {
+            "block_num": 5
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "value": "dGhpcyBpcyBhIGZpcnN0IHZhbHVlIHVwZGF0ZWQ=",
+        "metadata": {
+          "version": {
+            "block_num": 6
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      }
+    ]
+  },
+  "signature": "MEYCIQDoFCEmCAirIJHGcIi3/gz6Bd2a+iJdFvzIVRhXYg9RswIhANpSU319PofKI98lydpsjvz5IpIfvUJhKto8ISsrWfeR"
+}
+```
+
+As we can see here, `key1` changed his value three times, in blocks 4, 5, 6
+
+Now lets query `key3` history, using same GET query.
+
+```sh
+bin/signer -data '{"user_id":"alice","db_name":"db2","key":"key3"}' -privatekey=deployment/sample/crypto/alice/alice.key
+```
+
+**Output**
+```
+MEYCIQDwogJW4wuIBGO5MYVLt1F0nFq+6BsULpKpmCcyGrcx/QIhAMj1uhj6LLW14pF6zZGToP5rdd0tgx9fvwUDHLXLbDO+
+```
+
+Submit query
+```sh
+curl \
+     -H "Content-Type: application/json" \
+     -H "UserID: alice" \
+     -H "Signature: MEYCIQDwogJW4wuIBGO5MYVLt1F0nFq+6BsULpKpmCcyGrcx/QIhAMj1uhj6LLW14pF6zZGToP5rdd0tgx9fvwUDHLXLbDO+" \
+     -X GET http://127.0.0.1:6001/provenance/data/history/db2/key3 | jq .
+```
+
+**Output**
+
+```json
+{
+  "response": {
+    "header": {
+      "node_id": "bdb-node-1"
+    },
+    "values": [
+      {
+        "value": "dGhpcyBpcyBhIHRoaXJkIHZhbHVl",
+        "metadata": {
+          "version": {
+            "block_num": 5
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      }
+    ]
+  },
+  "signature": "MEQCIFE2JIa9MkaGtBEZUqx8MIsYsSSiJS0Atz+TkI6Yl73HAiBRU98GTF+FlQoVXvSuqZArwd5UxjK5aMXBJLEdRCQKTA=="
+}
+```
+
+Key `key3` changed only once, in block 5
+
+> Please note that due `json.Marshal` for protobuf objects limitations, in many cases inside `version` json object, only `block_num` field is presented, without `tx_num`, because `tx_num` equal to zero. But during unmarshalling protobuf object back from json, `tx_num` field filed will be assigned zero value.
```

Comment:
(limitations - not really, we can instruct the server to not omit default values in protobuf) 
due -> due to
equal to -> is equal to

Reply:
rewrote whole sentence.